src/pages/Books.js

```diff
@@ -2,11 +2,11 @@ import React from 'react';
 import Book from '../components/Book';
```

Comment:
- [ ] The Book (not Books) component should have a button for removing book from the store using the book ID.